src/DynamoCoreWpf/UI/Prompts/UsageReportingAgreementPrompt.xaml.cs

```diff
@@ -103,5 +86,15 @@ protected override void OnClosed(System.EventArgs e)
             base.OnClosed(e);
             viewModel = null;
         }
+
+        private void configure_adp_button_Click(object sender, RoutedEventArgs e)
+        {
+            IntPtr handle = new IntPtr();
```

Comment:
do we always need an owner handle ?
should this be initialized to null ?

Reply:
We `can` use null, but ADP dev suggested not leaving it null if possible.